src/node/crypto.ts

```diff
@@ -101,6 +101,14 @@ export {
   createSecretKey,
 }
 
+export function getHashes() {
+  // Hardcoded list of hashes supported in boringssl, node's approach looks pretty clunky. This is
+  // expected to change infrequently based of bssl's stability-focused approach.
+  return ['md4', 'md5', 'sha1', 'sha224', 'sha256', 'sha384', 'sha512', 'md5-sha1', 'RSA-MD5',
+          'RSA-SHA1', 'RSA-SHA224', 'RSA-SHA256', 'RSA-SHA384', 'RSA-SHA512', 'DSA-SHA',
+          'DSA-SHA1', 'ecdsa-with-SHA1'];
```

Comment:
This is fine. Are these all of the hashes supported by boringssl or just a subset?

Reply:
Yes, this includes all hashes as listed in https://boringssl.googlesource.com/boringssl/+/fips-20210429/crypto/digest_extra/digest_extra.c#77. boringssl also includes a number of additional deprecated hashes but they are not supported by `EVP_get_digestbyname` and thus won't be useful in a node crypto context.